superset-frontend/src/explore/components/ExploreChartHeader/index.jsx

```diff
@@ -235,8 +238,18 @@ export class ExploreChartHeader extends React.PureComponent {
       this.props.chart.chartStatus,
     );
     return (
-      <StyledHeader id="slice-header" className="panel-title-large">
+      <StyledHeader id="slice-header">
         <div className="title-panel">
+          <ChartEditableTitle
+            title={this.getSliceName()}
+            canEdit={
+              !slice ||
+              this.props.can_overwrite ||
```

Comment:
nit: while we're using the destructured `slice`, could we also destructure this like the others on line 224: `const { ..., can_overwrite: canOverwrite } = this.props`. Also maybe `this.props.actions.updateChartTitle`

Reply:
Done!